source/backend/plugin/CarlaPluginFluidSynth.cpp

```diff
@@ -1656,6 +1689,60 @@ class CarlaPluginFluidSynth : public CarlaPlugin
     }
 
 private:
+    void initializeFluidDefaultsIfNeeded()
+    {
+        if (!sbFluidDefaultsStored)
```

Comment:
please revert this check, like so:
```
if (sFluidDefaultsStored)
    return;
sFluidDefaultsStored = true;
// ...
```
The purpose is to continue the main code without extra indentation